homeassistant/components/ituran/sensor.py

```diff
@@ -33,6 +35,7 @@ class IturanSensorEntityDescription(SensorEntityDescription):
     """Describes Ituran sensor entity."""
 
     value_fn: Callable[[Vehicle], StateType | datetime]
+    is_available: Callable[[Vehicle], bool] = lambda _: True
```

Comment:
```suggestion
    supported_fn: Callable[[Vehicle], bool] = lambda _: True
```
otherwise it's too easy to mix it with entity availability, e.g. I was looking into where you're building your `available` property